includes/SiteConfiguration.php

```diff
@@ -1115,4 +1116,16 @@ public function getDatabaseConfig(): array
     {
         return $this->databaseConfig;
     }
+
+    public function getPrivacyStatementPath(): string
+    {
+        return $this->privacyStatementPath;
+    }
+
+    public function setPrivacyStatementPath(string $privacyStatementPath): SiteConfiguration
```

Comment:
Yes.

Thanks to [recent changes](https://github.com/enwikipedia-acc/waca/commit/f868cf7447ee54467d6b092dba8240060fbc60ee), the logic now looks like this: 

https://github.com/enwikipedia-acc/waca/blob/2acd7662e2a5f3f65c9cf9a9bf63bc33481717f2/includes/Startup.php#L18-L21

https://github.com/enwikipedia-acc/waca/blob/2acd7662e2a5f3f65c9cf9a9bf63bc33481717f2/config.inc.php#L181-L186

As such, my intention is to do this in config.local.inc.php (and this is how I tested it);

```php
global $siteConfiguration;
$siteConfiguration->setPrivacyStatementPath('/path/to/file.md');
```

The massive pile of fluent setters at the end of config.inc.php has been "temporary" since newinternal, and only recently have I figured out a sensible migration path to use SiteConfiguration directly. This'll be the first setting using it in that way.

Reply:
Ah neat, okay!